docs/reference/platforms/android/gradle.rst

```diff
@@ -266,6 +274,14 @@ will result in an ``AndroidManifest.xml`` declaration of::
 
     <uses-permission android:name="android.permission.HIGH_SAMPLING_RATE_SENSORS">
 
+``target_sdk_version``
+----------------------
```

Comment:
Even though there isn't any equivalent on the other platforms, would it be better to call this `target_os_version` to indicate that both settings are of the same type?

Reply:
Since it's an Android-specific setting, I was leaning towards preserving the "Android-ness" of it; but I given the comments below about how it's interpreted, maybe it makes sense to stay consistent with the other Briefcase setting.